packages/common/core-interfaces/src/jsonSerializationErrors.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Type resulting from {@link JsonSerializable} use given an array with
+ * `undefined` elements.
+ *
+ * @privateRemarks type is used over interface; so inspection of type
+ * result can be more informative than just the type name.
+ *
+ * @beta
+ * @system
```

Comment:
do we document `@system` user facing anywhere yet?

Reply:
We need to add support to the website. @Josmithr, want to give it a go? :) I did add notes to our wiki: https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#system

Reply:
Website support added with #22050